File: src/main/java/com/rtkit/upsource_manager/services/RoleService.java
package com.rtkit.upsource_manager.services;

import com.rtkit.upsource_manager.entities.user.Role;
import com.rtkit.upsource_manager.repositories.RoleRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.Collection;

@Service
public class RoleService {
	
	private final RoleRepository roleRepository;

	@Autowired
	public RoleService(RoleRepository roleRepository) {
		this.roleRepository = roleRepository;
	}
	/**
	 * Получение списка всех ролей
	 * @return
	 */
	public Collection<Role> findAll() {
		return roleRepository.findAll();
	}
}

File: src/main/java/com/rtkit/upsource_manager/services/RoleService.kt
package com.rtkit.upsource_manager.services

import com.rtkit.upsource_manager.entities.developer.Role
import com.rtkit.upsource_manager.repositories.RoleRepository
import org.springframework.stereotype.Service

@Service
class RoleService(
    private val roleRepository: RoleRepository
) {
    fun findAll(): MutableSet<Role> {
        return roleRepository.findAll().toMutableSet()
    }
}